projects/wasmtime/build.sh

```diff
@@ -26,6 +26,12 @@ build() {
   shift
   PROJECT_DIR=$SRC/$project
 
+  cd $PROJECT_DIR
```

Comment:
Could this have a comment for the purpose of what's happening? (basically just ensuring that absolute paths are used everywhere in debuginfo and such)